api/hub/src/datasources/moderation-admin.ts

```diff
@@ -23,7 +23,7 @@ class ModerationAdminAPI extends DataSource {
 
   /**
    * Get data for a moderator.
-   * @param address the ETH address of the moderator
+   * @param address The ETH address of the moderator
```

Comment:
it should also contain the `-` before the description like `@param address - The ETH address of the moderator`